ui/src/shared/reducers/appConfig.js

```diff
@@ -0,0 +1,20 @@
+import {AUTOREFRESH_DEFAULT} from 'src/shared/constants'
+
+const initialState = {
+  autoRefresh: AUTOREFRESH_DEFAULT,
+}
+
+const appConfig = (state = initialState, action) => {
```

Comment:
I would either move the stuff that's in appUI into appConfig or visa versa.  I created the appUI to hold global app state which is exactly what this is.  I don't think we need two separate reducers.